tox.ini

```diff
@@ -7,7 +7,7 @@ commands = bash .ci/flake8_wrapper.sh
 whitelist_externals = bash
 deps =
     flake8
-    flake8-docstrings==1.0.2
+    flake8-docstrings
```

Comment:
That's mostly because `flake8-docstrings` was pinned when introduced in #3004. I think it should stay un-pinned in `dev` as are the other deps.